code/datums/components/spawner.dm

```diff
@@ -1,14 +1,19 @@
 /datum/component/spawner
+	/// List of mob types to spawn, picked randomly
 	var/mob_types = list(/mob/living/basic/carp)
```

Comment:
Seems silly to have it default to carp (or default to anything actually)... might be wise to clean this up while we're trying to make this more stand-alone and not tied to any default behavior (as well as turn this into a `var/list` if that works)